.travis.yml

```diff
@@ -36,6 +36,8 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 7.2
+  - 7.3
```

Comment:
Hey @peter279k 

Because we don't have 7.2 and 7.3 in all of our testing, could you add these to an `allowed_failures` block as well.

Reply:
Hi @whikloj, thanks for your reply and suggestion.

I've let `php-7.2` and `php-7.3` versions be allowed failures.

You  can check that, thanks.